Community/gsoc-alumn.md

```diff
@@ -0,0 +1,108 @@
+# CHAOSS and Google Summer of Code Alumni
+
+## CHAOSS GSoC students in 2019
+
+|Student|Project|Mentors|
+|-------|:-----:|-------|
+|[Parth Sharma](#parth-sharma)|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins|
+|[Bingwen Ma](#bingwen-ma)|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins, Matt Germonprez|
+|[Aniruddha Karajgi](#aniruddha-karajgi)|Implementing CHAOSS Metrics with Perceval|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+|[Nishchith K Shetty](#nishchith-k-shetty)|Support of Source Code Related Metrics|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+
+## CHAOSS GSoC students in 2018
+
+|Student|Project|Mentors|
+|-------|:-----:|-------|
+|[Keanu Nichols](#keanu-nichols)|Reporting of CHAOSS Metrics|Sean Goggins, Jesus Gonzalez-Barahona|
+|[Pranjal Aswani](#pranjal-aswani)|Reporting of CHAOSS Metrics: Refactoring the existing code and extending the capabilities of the Manuscripts Project|Valerio Cosentino, Jesus Gonzalez-Barahona|
+
+
+
+## Parth Sharma
+
+### Project Title 
+Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur
+### Project Description
+Augur is fully functional prototyping web stack for CHAOSS metrics. It provides structured data mined from git repositories using a plugin architecture that incorporates other open source metrics projects like Facade and FOSSology. The main aim of this project is to extend Augur’s functionality by implementing Risk and Growth-Maturity-Decline CHAOSS metrics and use cases with a focus on the open source community manager use case. This project, with a focus on the community manager use case, will allow open source community managers to leverage Risk and Growth-Maturity-Decline metrics to better manage their communities and projects.
+
+### Blog posts
+[Weekly Posts](https://parthsharma2.github.io/posts)
+
+### Social Media
+[LinkedIn](https://www.linkedin.com/in/parthsharma2)
+[Twitter](https://twitter.com/parthshar2)
```

Comment:
I think it would look better if these were lists.

```suggestion
* [LinkedIn](https://www.linkedin.com/in/parthsharma2)
* [Twitter](https://twitter.com/parthshar2)
```